### PR TITLE
Adds Advertisers.all [#104683988]

### PR DESCRIPTION
**Pull request by padi**
http://pivotaltracker.com/story/show/104683988

### CODE REVIEW

File: lib/yahoo_gemini_client/requests/advertisers_fetch_request.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+    def execute
+      response = get(request_uri: build_request_uri)
+      AdvertisersResponse.new(response.with_indifferent_access)
+    end
+
+    private
+
+    def build_request_uri
+      REQUEST_URI
```
Comment (jpmoral) [inline lib/yahoo_gemini_client/requests/advertisers_fetch_request.rb]: Where is `REQUEST_URI` defined?

Comment (padi) [inline lib/yahoo_gemini_client/requests/advertisers_fetch_request.rb]: https://github.com/G5/yahoo_gemini_client/blob/master/lib/yahoo_gemini_client/requests/advertiser_base_request.rb#L3

**Comment by ramontayag** [comment]
:ship:

**Comment by Mau3189** [comment]
Hey guys, is this working on version `0.1.1`?

**Comment by ramontayag** [comment]
@Mau3189 - nope, it is not: https://github.com/G5/yahoo_gemini_client/blob/master/CHANGELOG.md

**Pull request merged**